docs/topics/third-party-packages.md

```diff
@@ -285,7 +285,6 @@ To submit new content, [open an issue][drf-create-issue] or [create a pull reque
 [discussion-group]: https://groups.google.com/forum/#!forum/django-rest-framework
 [djangorestframework-digestauth]: https://github.com/juanriaza/django-rest-framework-digestauth
 [django-oauth-toolkit]: https://github.com/evonove/django-oauth-toolkit
-[doac]: https://github.com/Rediker-Software/doac
```

Comment:
The corresponding doac ref on line no 186 should be removed.

Reply:
Thanks, missed that! I was grepping for the wrong thing.